lib/browser.ts

```diff
@@ -775,8 +775,8 @@ export class ProtractorBrowser extends Webdriver {
         .then(loadMocks, deferred.reject);
 
     let self = this;
-    function loadMocks(angularVersion: number) {
-      if (angularVersion === 1) {
+    function loadMocks(angularVersion: string) {
+      if (angularVersion == '1') {
```

Comment:
But why must `loadMocks` have a string value? Because it's inferred from the type of the `angularTestResult` variable whose type is wrong.